src/content/docs/auto-telemetry-pixie/manage-pixie-data.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Manage your Auto-telemetry with Pixie data ingest
+tags:
+  - Pixie Auto-telemetry
+  - Kubernetes pods
+  - Kubernetes
+  - manage Pixie data
+metaDescription: 
+redirects:
+ 
+---
+
+You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce or exclude data, either by excluding specific namespaces or pods, or by collecting data for only the nodes you want.
+
+## Exclude namespaces and pods [#reduce-ingest-helm]
+
+If you want to reduce the amount of Pixie data that New Relic ingests, you can exclude namespaces or pods by adding the following parameters to your Helm chart during installation. Note that the data still exists in Pixie:
+
+  * `excludeNamespacesRegex` - use to identify the namespaces that you want to exclude from sending observability data to New Relic. If empty, data for all namespaces is sent to New Relic. For example: 
+  
+  ```
+  --set newrelic-pixie.excludeNamespacesRegex="examplenamespace-1|examplenamespace-2"
+  ```
+  * `excludePodsRegex` - use to identify pods across all namespaces that you want to exclude from sending observability data to New Relic. If empty, data for all pods (except those in excluded namespaces) is sent to New Relic. Example: `--set newrelic-pixie.excludePodsRegex="examplepod-1|examplepod-2"`
+  
+These two configuration options provide additional control over the metric and span data sent to New Relic from Pixie.   
```

Comment:
Metric and Span should be capitalized as these are official event namespaces in New Relic.